.scripts/tag.sh

```
@@ -0,0 +1,37 @@
+set -u
+
+BOLD=$(tput bold)
+NORMAL=$(tput sgr0)
+
+echo "${BOLD}Tagging${NORMAL}"
+
+#get highest tag number
+VERSION=`git describe --abbrev=0 --tags`
+
+#replace . with space so can split into an array
+VERSION_BITS=(${VERSION//./ })
```

Shellcheck

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)